### PR TITLE
feat(patch9): Added Patch 9

### PR DESCRIPTION
**Pull request by viv9k**
Signed-off-by: vivek kumar <vvksindia@gmail.com>

### Functionality:
Added Patch 9 to update field and value in users column

### Solution:
Added Patch 9

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
Check in route patch 9

### CODE REVIEW

**Review by Pendyala-thanvi — COMMENTED** [review]
Resolve merge conflicts

File: functions/model/patch/patch.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ const { patchModerator } = require("./patchModerator");
 const { editPatch } = require("./editPatch");
 const { patch6 } = require("./patch6");
 const { patch7 } = require("./patch7");
+<<<<<<< HEAD
+const { patch9 } = require("./patch9");
+=======
 const { patch8 } = require("./patch8");
+>>>>>>> aaa16e06be9d2ee3a3083000fbddaffe1e7f2b11
```
Comment (arjund-06) [inline functions/model/patch/patch.js:23]: Merge conflict not solved properly

File: functions/model/patch/patch.js [hunk 2]
```diff
@@ -43,9 +47,15 @@ exports.patch = functions.https.onRequest((request, response) => {
...
             return patch7(request, response);
+<<<<<<< HEAD
+        } else if (mode == "patch9") {
+            // To add a new field to the organization
+            return patch9(request, response);
+=======
         } else if (mode == "patch8") {
             // To add a new field and value to teams
             return patch8(request, response);
+>>>>>>> aaa16e06be9d2ee3a3083000fbddaffe1e7f2b11
```
Comment (arjund-06) [inline functions/model/patch/patch.js:58]: Merge conflict not solved properly

File: functions/model/patch/patch.js (hunk 1 above)
Comment (viv9k) [reply functions/model/patch/patch.js]: resolved this issue

File: functions/model/patch/patch.js (hunk 2 above)
Comment (viv9k) [reply functions/model/patch/patch.js]: resolved this issue

**Pull request merged**